api/init/proxy/interceptor.py

```diff
@@ -1,7 +1,9 @@
 import sys
 from graphql.ast import Document
-import proxy.batch as batch  # Called dynamically with getattr pylint: disable=W0611, useless-import-alias
-import proxy.data_source as data_source  # Called dynamically with getattr pylint: disable=W0611, useless-import-alias
+# Called dynamically with getattr pylint: disable=W0611, useless-import-alias
+from proxy import batch
```

Comment:
Have you tested this still works with the getAddr Code?

Reply:
I didn't test it, but according to my knowledge of Python modules, it shouldn't make a difference how you import modules.

Reply:
Ok, just tested it. It still works.